generate/main.go

```diff
@@ -23,17 +23,19 @@ import (
 
 // must be sorted
 var ignoredFields = []string{
-	"aclid",
-	"acltype",
+	"customid",
+	"deploymentplanner",
+	"displaynetwork",
+	"displayvm",
+	"displayvolume",
 	"forvpc",
+	"haenable",
+	"hostid",
+	"hypervisor",
+	"iscallerchilddomain",
+	"isdynamicallyscalable",
+	"podid",
```

Comment:
these field `domainid,domain,regionid,listall,isrecursive...` have not to be here? I don't see it

Reply:
My bad they are not anymore in  the listApi